dns/ddclient/src/opnsense/scripts/ddclient/lib/account/cloudflare.py

```diff
@@ -142,7 +142,6 @@ def execute(self):
                 return False
 
             record_id = payload['result'][0]['id']
-            proxied = payload['result'][0]['proxied']
```

Comment:
this doesn't look intentional.

Reply:
Ok, sounds logical, their documentation isn't very clear on the subject by the way, but lets merge and see what happens.